### DISCUSSION TITLE
Difference in recommended Phishing email threshold

### DISCUSSION THREAD

**Discussion by GittyHubGoGo** [cammurray/orca#259] [Q&A] [answered]
Hi, 

I have a question regarding the Phishing email threshold settings which are recommended. 
Looking at https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/recommended-settings-for-eop-and-office365?view=o365-worldwide#advanced-settings-in-anti-phishing-policies-in-microsoft-defender-for-office-365 it states that the recommended threshold for standard is 3 where for strict policy it is 4. 

When running ORCA version 2.5 it states that the threshold should be 2 or 3. 
What recommendation to follow and is the difference expected?
Tnx. 

![MDO 365 Recommended Configuration Analyzer](https://github.com/cammurray/orca/assets/144007486/342f24e0-9618-4b23-a8ef-f89701cbab40)

**Comment by cammurray** [comment, marked as answer]
Hey @GittyHubGoGo , these changes to the recommendations were only made recently, and ORCA 2.6 has been updated here: #252 

We are having a few issues at the moment with getting the change uploaded to the PowerShell gallery. Hopefully will be resolved soon, in the mean time just ensure you’re at 3/4
